crates/ol/block-assembly/src/context.rs

```diff
@@ -0,0 +1,66 @@
+//! Block assembly context trait.
+
+use strata_asm_common::AsmLogEntry;
+use strata_ol_chain_types_new::{OLBlockId, OLTransaction};
+use strata_ol_mempool::MempoolIndex;
+use strata_ol_state_types::OLState;
+use strata_primitives::l1::L1BlockCommitment;
+
+use crate::error::BlockAssemblyError;
+
+/// Context interface for block assembly.
+pub trait BlockAssemblyContext {
+    /// Gets parent block's [`OLState`].
```

Comment:
```suggestion
    /// Gets block's [`OLState`] corresponding to a block id.
```
Because the function doesn't say anything about the parent.